Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM alpine:latest AS builder
```

Comment:
I would use a fixed version of alpine instead of `latest` to prevent possible breaks. For instance, when `latest` will use Python 3.7, `/usr/lib/python3.6/` used in the recipe won't exist.

Reply:
Yeah, was thinking whether I should fix the version or not. I think you're right, it's better to fix. Done along with a couple of other improvements. Please let me know what you think about it now, when you have time.